src/schema/v2/verify_address.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLUnionType,
+  GraphQLString,
+  GraphQLEnumType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import {
+  GravityMutationErrorType,
+  formatGravityError,
+} from "lib/gravityErrorHandler"
+
+const ErrorType = new GraphQLObjectType<any, ResolverContext>({
+  name: "VerifyAddressMutationFailure",
+  isTypeOf: (data) => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => (typeof err.message === "object" ? err.message : err),
+    },
+    statusCode: {
+      type: GraphQLInt,
+      resolve: (result) => result.statusCode,
+    },
+    message: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: () => "Address could not be verified.",
+    },
+  }),
+})
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "VerifyAddressMutationSuccess",
+  isTypeOf: (data) => data.success,
+  fields: () => ({
+    verificationStatus: {
+      type: new GraphQLEnumType({
+        name: "VerificationStatuses",
+        values: {
+          VERIFIED_NO_CHANGE: { value: "VERIFIED_NO_CHANGE" },
+          VERIFIED_WITH_CHANGES: { value: "VERIFIED_WITH_CHANGES" },
+          VERIFICATION_UNAVAILABLE: { value: "VERIFICATION_UNAVAILABLE" },
+          NOT_FOUND: { value: "NOT_FOUND" },
+        },
+      }),
+      resolve: (result) => result.verificationStatus,
+    },
```

Comment:
For reference, possible values from gravity are defined [here](https://github.com/artsy/gravity/blob/main/app/services/address_verification_service/constants.rb#L13-L19). The not performed case should never happen when queried by the api because it always attempts verification.

I'm not clear how GraphQLEnums react to an unexpected value coming through. Is there anything we need to do to provide a fallback in case the api changes and we don't prepare this field?